scarb/src/ops/compile.rs

```diff
@@ -4,12 +4,11 @@ use cairo_lang_compiler::diagnostics::DiagnosticsError;
 use cairo_lang_utils::Upcast;
 use indoc::formatdoc;
 use itertools::Itertools;
-use smol_str::{SmolStr, ToSmolStr};
-use std::thread;
-
 use scarb_ui::args::FeaturesSpec;
 use scarb_ui::components::Status;
 use scarb_ui::HumanDuration;
+use smol_str::{SmolStr, ToSmolStr};
+use std::thread;
```

Comment:
irrelevant changes

Reply:
Probably an autoformatter, will leave it as it is